lib/widgets/lightbox.dart

```diff
@@ -492,11 +493,25 @@ class _VideoLightboxPageState extends State<VideoLightboxPage> with PerAccountSt
     _controller?.removeListener(_handleVideoControllerUpdate);
     _controller?.dispose();
     _controller = null;
+    WakelockPlus.disable();
     super.dispose();
   }
 
-  void _handleVideoControllerUpdate() {
-    setState(() {});
+  Future<void> _handleVideoControllerUpdate() async {
+    if (mounted) setState(() {});
```

Comment:
```
lightbox: Toggle wakelock while video is playing
```

The `if (mounted)` condition is new here. It looks reasonable, I think, but is it part of "Toggle wakelock while video is playing"? It looks orthogonal and like the kind of thing that belongs in a prep commit.